src/domain/types.ts

```diff
@@ -0,0 +1,7 @@
+export enum CUSTOMER_GROUP {
```

Comment:
nit: Looks like this is otherwise a one to one mapping with `OrganizationType`, except for the addition of `PRIVATE` if no organization type is returned from the backend. Would it rather make sense to have this information in the backend? If the backend enum is changed (additions/deletions) this and the mapper need to be kept in sync. There might be some architectural decision behind this that I'm not aware of.

Reply:
I agree I think we should rely on the `OragnizationType` `enum` generated from GQL schema.

Reply:
I removed customer group mapping and changed the code to use OrganizationType. If it's null or undefined, the customer is considered private.